### ISSUE TITLE
Add AWS_PROFILE settings

### ISSUE THREAD

**Issue by witold-gren** [jschneier/django-storages#774]
If you work in a few projects better way (locally) is usage AWS profiles (`IAM Roles`) configured in awscli. You usage one credentials account (`AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`) and much more AWS profiles.

**Comment by witold-gren** [comment]
Actually it's a better solution is usage `sts` client to start usage IAM Role without configuring `awscli`, We must only set two extra params `AWS_ROLE_ARN ` and `AWS_ROLE_SESSION_NAME `:
```
client = boto3.client('sts')
response = client.assume_role(
    RoleArn= AWS_ROLE_ARN, 
    RoleSessionName= AWS_ROLE_SESSION_NAME)
session = boto3.Session(
    aws_access_key_id=response['Credentials']['AccessKeyId'],
    aws_secret_access_key=response['Credentials']['SecretAccessKey'],
    aws_session_token=response['Credentials']['SessionToken'])
```

**Comment by jschneier** [comment]
Fixed by #919
